packages/vscode-java-tools/v0.29.2/setup.d/01-setup-extensions.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+set -eo pipefail
+
+PACKAGE_DIR=$(cd "$(dirname $BASH_SOURCE)/.."; pwd)
+
+# Install extensions
+code-server --install-extension $PACKAGE_DIR/extensions/redhat.java-amd64/java-linux-x64-1.42.0-561.vsix
+code-server --install-extension $PACKAGE_DIR/extensions/redhat.java-arm64/java-linux-arm64-1.42.0-561.vsix
```

Comment:
Does it harm or there's no issue with having both amd and arm64 vsix files in here? Probably the script could have an if statement checking for the $PLATFORM_ARCH environment variable and install only the appropriate

Reply:
Good catch.

This should be parameterized based on $PLATFORM_ARCH.

See last commit.